src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -262,6 +263,17 @@ public class JdbcSinkConnectorConfig {
             .withDefault(false)
             .withDescription("Allowing to insert explicit value for identity column in table for SQLSERVER.");
 
+    public static final Field BATCH_SIZE_FIELD = Field.create(BATCH_SIZE)
+            .withDisplayName("Specifies how many records to attempt to batch together into the destination table, when possible. " +
+                    "You can also configure the connector’s underlying consumer’s max.poll.records using consumer.override.max.poll.records in the connector configuration.")
+            .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 4))
+            .withWidth(ConfigDef.Width.SHORT)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .withDefault(500)
```

Comment:
I believe we should likely increase this default value to something a bit more OOTB production-ready, perhaps either `2500`, `3000`, or `5000`.  The Confluent one defaults to `3000` per their documentation.

Reply:
Agree that `500` is low. I would like to do some performance test to find a better default value.

Reply:
Just a note, this number is the same as the `consumer.max.poll.records` default. With the current buffering implementation maybe its better to leave `batch.size` at the same default value and document the flushing behavior. WDYT?

Reply:
Yes, let's keep it as is. We can always reconsider it.